nuklear.h

```diff
@@ -23475,78 +23590,117 @@ nk_handle_id(int id)
     return handle;
 }
 NK_API struct nk_image
-nk_subimage_ptr(void *ptr, nk_ushort w, nk_ushort h, struct nk_rect r)
+nk_subimage_from_ptr(void *ptr, nk_ushort w, nk_ushort h, enum nk_image_type type, struct nk_rect r)
 {
     struct nk_image s;
     nk_zero(&s, sizeof(s));
     s.handle.ptr = ptr;
+    NK_ASSERT(w); 		/* Size may not be zero */
+    NK_ASSERT(h); 		/* Size may not be zero */
```

Comment:
I think there are use cases for zero size(s). Right now I can think of only the following one but I bet there are more.

1. An image in JPEG, PNG, etc. formats is allowed to have size 0x0 if I am not mistaken. Loading such an image **must not** crash (on assert or otherwise) the app.

Reply:
Fixed